javaagent-bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/AgentInitializer.java

```diff
@@ -159,7 +159,7 @@ private static synchronized void installAgentTracer() {
     try {
       // install global tracer
       Class<?> tracerInstallerClass =
-          AGENT_CLASSLOADER.loadClass("io.opentelemetry.javaagent.tooling.TracerInstaller");
+          AGENT_CLASSLOADER.loadClass("io.opentelemetry.javaagent.tooling.OpenTelemetryInstaller");
```

Comment:
it feels a little odd to use the OpenTelemetryInstaller class as both an SPI interface and as a static entry point here

Reply:
the PR is not ready yet :)

Reply:
I have updated the PR, this has been removed from the agentInitializer